data/common/apache.yaml

```diff
@@ -0,0 +1,45 @@
+---
+
+lookup_options:
+  ducktape::apache::confs:
+    merge:
+      strategy: deep
+      knockout_prefix: --
+
+ducktape::apache::conf_defaults: {}
+ducktape::apache::confs: {}
+ducktape::apache::enabled: true
+ducktape::apache::autoload::autoload: true
+
+ducktape::apache::conf::ensure: 'present'
+ducktape::apache::conf::priority: '010'
+ducktape::apache::conf::directives: []
+ducktape::apache::conf::content: ~
+
+ducktape::apache::shield_vhost::enabled: true
+ducktape::apache::shield_vhost::ensure: 'present'
+ducktape::apache::shield_vhost::port: 80
+#ducktape::apache::shield_vhost::docroot: "%{lookup('apache::docroot')}"
```

Comment:
Este esto correcto?

Reply:
I think no since `apache::docroot` might be defined on code instead of hiera

Reply:
Well for now for these particular cases, when the value of a parameter is a puppet resource/class param, I have left their default values inside the puppet code.

Reply:
Déjalo en la clase y crea un ticket en team para ver si podemos mejorar o simplificar la clase shield_vhost.